invenio_rdm_records/services/services.py

```diff
@@ -191,3 +193,44 @@ def discard_pid(self, id_, identity, pid_type, pid_client=None):
             draft,
             links_tpl=self.links_item_tpl,
         )
+
+    def lift(self, _id, identity):
```

Comment:
what about `lift_embargo`, I understand is longer but otherwise might not be clear that it means embargo